libmscore/cmd.cpp

```diff
@@ -2224,8 +2224,10 @@ Element* Score::move(const QString& cmd)
       else
             cr = selection().lastChordRest();
 
-      // no chord/rest found? look for another type of element
-      if (cr == 0) {
+      // no chord/rest found? look for another type of element,
+      // but commands [empty-trailing-measure] and [top-staff] don't
+      // necessarily need an active selection for appropriate functioning
+      if (cr == 0 && cmd != "empty-trailing-measure" && cmd != "top-staff") {
```

Comment:
the proposed top and trailing measure functions here have been given functionality without needing a selection, yet still operate within the framework of Score::move() where move selections occur (which is where these commands fit right in), so i allow them to fall through even if no valid chord rest was found here

Reply:
I'd rather use `if (!cr && ...` here

Reply:
So would I

Reply:
There are a handful of cr == 0 statements in this file, so to make the change only here would be slightly absurd without changing the rest. I'll do a force push shortly